src/main/java/org/sonarsource/scanner/cli/Main.java

```diff
@@ -65,22 +65,26 @@ void execute() {
     Stats stats = new Stats(logger).start();
 
     int status = Exit.ERROR;
+    boolean started = false;
     try {
       Properties p = conf.properties();
       checkSkip(p);
       configureLogging(p);
       init(p);
       runner.start();
+      started = true;
       logger.info("SonarQube server " + runner.serverVersion());
       runAnalysis(stats, p);
       status = Exit.SUCCESS;
-    } catch (Exception e) {
+    } catch (Throwable e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1181)